collect_monitoring.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Define SFTP server details
+SFTP_SERVER="sftp.ams20.com"
+SFTP_USERNAME="tta_ro"
```

Comment:
Maybe I just haven't come across it yet, but where do we find the username for this? I assume this is an FTP user tied to the docker container, not the sandbox app?

Reply:
This has been refactored to have to all the ftp connection settings defined indirectly via ENV. This now also allows the port for the ftp server to be changed.